pkg/controllers/machine/disruption/drift.go

```diff
@@ -24,12 +24,19 @@ import (
 	"knative.dev/pkg/logging"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	"github.com/samber/lo"
+
 	"github.com/aws/karpenter-core/pkg/apis/settings"
 	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
 	"github.com/aws/karpenter-core/pkg/cloudprovider"
 	"github.com/aws/karpenter-core/pkg/scheduling"
 )
 
+const (
+	StaticDrift          cloudprovider.DriftReason = "StaticDrift"
+	NodeRequirementDrift cloudprovider.DriftReason = "NodeRequirementDrift"
+)
```

Comment:
Consider making an enum for not drifted for readability
```suggestion
const (
	StaticDrift          cloudprovider.DriftReason = "StaticDrift"
	NodeRequirementDrift cloudprovider.DriftReason = "NodeRequirementDrift"
	NotDrifted         cloudprovider.DriftReason = "NotDrifted" || ""
)
```